src/editor/AtfMode.js

```diff
@@ -0,0 +1,40 @@
+import 'brace/mode/java'
+
+export class AtfHighlightRules extends window.ace.acequire(
+  'ace/mode/text_highlight_rules'
+).TextHighlightRules {
+  constructor() {
+    super()
+    this.$rules = {
+      start: [
+        {
+          token: 'comment',
+          regex: '^@.*$'
+        },
+        {
+          token: 'constant.language.boolean',
+          regex: '^\\$.*$'
+        },
+        {
+          token: 'constant.numeric',
+          regex: '^#.*$'
+        },
+        {
+          token: 'string',
+          regex: '\\|[^\\|]*\\|'
+        },
+        {
+          token: 'support.function',
```

Comment:
The colour is very close to the text colour in the current theme. 

The token types seem to be semantically wrong. Unfortunately, Ace was meant to be a code editor and tokens match that domain. While it is impossible to get 100% "correct" tokens we should try to find meaningful values. See: [Common Tokens](https://github.com/ajaxorg/ace/wiki/Creating-or-Extending-an-Edit-Mode#commonTokens). Ideally, we should create our own theme, but that's probably worth its own card.

Suggestions for tokens:

- **#**: `comment.line.number`
- **@**: `keyword` or `meta`. These lines mark on which side, column etc. the text is on.
- **$**: `markup`. These lines mark underlinings, blank lines etc.